xml/admin_ceph_gateway.xml

```diff
@@ -3152,6 +3165,43 @@ data sync source: 341c2d81-4574-4d08-ab0f-5a2a7b168028 (us-east)
                   full sync: 0/128 shards
                   incremental sync: 128/128 shards
                   data is caught up with source</screen>
+    <para>
+     The output can differ depending on the sync status. The shards are
+     described as two different types during sync:
+    </para>
+    <variablelist>
+     <varlistentry>
+      <term>Behind shards</term>
+      <listitem>
+       <para>
+        Behind shards are shards that need a full data sync and shards needing
+        an incremental data sync because they are not up-to-date.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>Recovery shards</term>
+      <listitem>
+       <para>
+        Recovery shards are shards that encountered an error during sync and
```

Comment:
```suggestion
        Recovery shards encounter errors during sync and

Reply:
And again: 

«
Shards described as [emphasis]recovery[/emphasis] encountered an error during sync and are marked to retry. This error mostly occurs...
»